exonum/src/node/state.rs

```diff
@@ -518,8 +519,9 @@ impl State {
             .iter()
             .position(|pk| pk.consensus_key == *self.consensus_public_key())
             .map(|id| ValidatorId(id as u16));
-        self.whitelist
-            .set_validators(config.validator_keys.iter().map(|x| x.consensus_key));
+
+        self.refresh_connect_list(&config.validator_keys);
```

Comment:
Do I understand correctly that this method will drop all auditors from the connect list? Is this intended?